interactive_ai/workflows/dataset_ie/job/utils/import_utils.py

```diff
@@ -169,14 +169,13 @@ def get_validated_labels_map(labels_map: dict, project_identifier: ProjectIdenti
         :param project_identifier: Identifier of the project containing the labels
         :return: The validated labels map which maps the datumaro label name to the project's label
         """
-
         valid_labels_map = {}
         label_repo = LabelRepo(project_identifier)
         for dm_name, sc_label_id in labels_map.items():
             label = label_repo.get_by_id(ID(sc_label_id))
             if isinstance(label, NullLabel):
                 raise InvalidLabelException(f"The provided label ID({sc_label_id}) does not belong to any label.")
-            valid_labels_map[dm_name] = label
+            valid_labels_map[str(dm_name)] = label
```

Comment:
If `dm_name` is not a string, then the problem is in the construction of `labels_map`. Improve the type-hint of `labels_map` to `dict[str, str]` to catch such issues.